toponetx/__init__.py

```diff
@@ -1,32 +1,26 @@
 __version__ = "0.0.2"
 
 from toponetx.exception import (
-    TopoNetXException,
     TopoNetXError,
+    TopoNetXException,
     TopoNetXNotImplementedError,
 )
 
-from .classes.complex import Complex
-from .classes.simplicial_complex import SimplicialComplex
+from .classes.cell import Cell
 from .classes.cell_complex import CellComplex
 from .classes.combinatorial_complex import CombinatorialComplex
-
-from .classes.simplex import Simplex
+from .classes.complex import Complex
 from .classes.hyperedge import HyperEdge
-from .classes.cell import Cell
```

Comment:
It seems these imports are not sorted alphabetically because they are grouped by function. Someone must have made this decision, and I can see that it may actually be easier to find imports this way.

Overall, I'm not sure if we should go ahead with this PR or keep it the way it was.

Reply:
Since only one line is shown in the preview, here is a larger excerpt from the code

```python
from .classes.complex import Complex
from .classes.simplicial_complex import SimplicialComplex
from .classes.cell_complex import CellComplex
from .classes.combinatorial_complex import CombinatorialComplex

from .classes.simplex import Simplex
from .classes.hyperedge import HyperEdge
from .classes.cell import Cell
```